src/confluent_kafka/admin/__init__.py

```diff
@@ -1204,3 +1215,36 @@ def list_offsets(self, topic_partition_offsets, **kwargs):
 
         super(AdminClient, self).list_offsets(topic_partition_offsets_list, f, **kwargs)
         return futmap
+
+    def delete_records(self, topic_partition_offsets_list, **kwargs):
```

Comment:
Lets use `topic_partition_offsets` instead of `topic_partition_offsets_list` to be consistent with the other Admin APIs